src/Simulation/Simulators/QuantumSimulator/StateDumper.cs

```diff
@@ -60,29 +67,190 @@ public virtual bool Dump(IQArray<Qubit>? qubits = null)
         }
 
         /// <summary>
-        /// A simple implementation of a <see cref="StateDumper"/>. It outputs the
-        /// a string representation of the state to the given channel.
+        ///     The convention to be used in labeling computational basis states
+        ///     given their representations as strings of classical bits.
         /// </summary>
-        public class SimpleDumper : StateDumper
+        [JsonConverter(typeof(StringEnumConverter))]
```

Comment:
Can you explain the meaning of this in a comment?

Reply:
This code has been moved from IQSharp repo. 
@cgranade, can you tell what that fragment means?

Reply:
This fragment is a fairly standard JSON serialization attribute that specifies enum values are converted as strings, not as ints.